content/docs/command-reference/plots/index.md

```diff
@@ -64,10 +64,10 @@ epoch, AUC, loss
 37, 0.92302, 0.0299015
 ```
 
-In hierarchical file formats such as JSON, an array of JSON objects is expected.
-`dvc plots show` command can generate visuals for a specified field name or a
-set of fields from the array's object. Like `val_loss` field in the `train`
-array in this example:
+In hierarchical file formats (JSON or YAML), an array of consistent objects is
```

Comment:
`consistent` - let's be explicit here

Reply:
Added "every object should have the same structure" for now. But actually I'm not sure what the exact restriction is. If one object has some extra keys maybe it won't impact plots as long as you don't try to use those keys? Cc @pared

> See dc0725cb

Reply:
Let's extract as a thing to check

Reply:
The fastest way for this - is to test probably